wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandScheduleTest.java

```diff
@@ -116,4 +118,27 @@ void notScheduledCancelTest() {
       assertDoesNotThrow(() -> scheduler.cancel(mockCommand));
     }
   }
+
+  @Test
+  void smartDashboardCancelTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      var inst = NetworkTableInstance.create();
```

Comment:
Should this be closed at the end of the test (in the try-with-resources)?

Reply:
Probably a good idea.

Reply:
Is this resolved? I think Starlight might have meant to include the nt instance in the try-with-resources specification to guarantee it will be closed, even in an exception (see below).
```diff
-    try (CommandScheduler scheduler = new CommandScheduler()) {
-      var inst = NetworkTableInstance.create();
+    try (CommandScheduler scheduler = new CommandScheduler(); var inst = NetworkTableInstance.create()) {
...
-      inst.close();
```

Reply:
Didn't know you could do that. Done now.